package.json

```diff
@@ -44,13 +44,12 @@
     "license": "GPL-3.0-only",
     "private": false,
     "devDependencies": {
-        "@nestjs/common": "^10.3.8",
-        "@nestjs/core": "^10.3.8",
+        "@nestjs/common": "^11.1.1",
+        "@nestjs/core": "^11.1.1",
         "@typegoose/typegoose": "^12.5.0",
-        "@types/mongoose": "^5.11.97",
```

Comment:
The `mongoose` package comes with built-in types, which is why I removed `@types/mongoose` altogether.